flycs_sdk/pipelines.py

```diff
@@ -340,19 +348,28 @@ def _is_valid_trigger(trigger: PipelineTrigger) -> bool:
 
 # sine we support python3.6 we cannot use datetime fromisoformat and isoformat methods
 # instead we use this
-_time_format = "%Y-%m-%dT%H:%M:%S%z"
+_time_format = "YYYY-MM-DDTHH:mm:ss"
 
 
 def _format_datetime(t: datetime) -> str:
     return t.strftime(_time_format)
 
 
-def _parse_datetime(tstr: str) -> datetime:
-
-    # ensure we always have the UTC timezone information
-    if not tstr.endswith("+0000"):
-        tstr += "+0000"
-    return datetime.strptime(tstr, _time_format)
+def _parse_datetime(tstr: str, timezone: Optional[str] = "UTC") -> pendulum.DateTime:
+    """Parse a pendulum datetime with its specific timezone from a string date."""
+    if "+" in tstr:
+        tstr = tstr.split("+")[0]
+        DeprecationWarning(
+            "Adding +0000 to specify timezone is decrepated. Now use format start_time format like 2021-09-18T00:00:00. To specify the timezone use 'timezone' attribute"
+        )
+    try:
+        tmz = pendulum.timezone(timezone)
+        dt = pendulum.from_format(tstr, _time_format, tz=tmz)
+        return dt
+    except:
+        raise Exception(
+            "Wrong pendulum datetime configuration, please make sur you're using one of the following timezone name : https://en.wikipedia.org/wiki/List_of_tz_database_time_zones"
```

Comment:
End user has no idea what pendulum is, this is confusing. 
What are the possible exceptions that can be raised here ?